javascript-practice/src/js/services/userService.js

```diff
@@ -76,6 +77,10 @@ export default class UserService extends CommonService {
 
     // Add token to user object
     newUserData.accessToken = createToken();
+
     this.save(newUserData);
+
+    // Add access token to local storage
+    LocalStorageService.add('accessToken', newUserData.accessToken);
```

Comment:
For storage key or fixed value, consider to create new constant pls

Reply:
Fixed.